spec/lib/telephony/telephony_spec_helper.rb

```diff
@@ -0,0 +1,61 @@
+require 'i18n/tasks'
+require 'pry-byebug'
+require 'webmock/rspec'
+require 'telephony/telephony'
```

Comment:
I don't think we need this file. Maybe we can move `use_default_config!` to like a `use_default_telephony_config!` and put it in the support directory